src/v2/Components/NavBar/Menus/NotificationsMenu.tsx

```diff
@@ -2,24 +2,58 @@ import { AnalyticsSchema } from "v2/Artsy/Analytics"
 import { useTracking } from "v2/Artsy/Analytics/useTracking"
 import React, { useContext } from "react"
 import { graphql } from "react-relay"
-
 import { SystemContext } from "v2/Artsy"
 import { get } from "v2/Utils/get"
-
-import {
-  LoadProgressRenderer,
-  renderWithLoadProgress,
-} from "v2/Artsy/Relay/renderWithLoadProgress"
-
+import { LoadProgressRenderer } from "v2/Artsy/Relay/renderWithLoadProgress"
 import {
   NotificationsMenuQuery,
   NotificationsMenuQueryResponse,
 } from "v2/__generated__/NotificationsMenuQuery.graphql"
-
-import { Box, Flex, Image, Link, Sans, Separator, Serif } from "@artsy/palette"
-import { Menu, MenuItem } from "v2/Components/Menu"
+import {
+  Box,
+  Image,
+  Link,
+  Separator,
+  SkeletonBox,
+  SkeletonText,
+  Text,
+} from "@artsy/palette"
+import { Menu, MenuItem, MenuItemPlaceholder } from "v2/Components/Menu"
 import { SystemQueryRenderer as QueryRenderer } from "v2/Artsy/Relay/SystemQueryRenderer"
 
+type NotificationMenuItemContentProps = {
+  image: JSX.Element
+  summary: JSX.Element
+  artists: JSX.Element
+}
```

Comment:
Small non-blocking consistency nit - we use interfaces everywhere else, and generally keep `type`s to one-liners